solid-notifications.ttl

```diff
@@ -0,0 +1,96 @@
+@prefix vs: <http://www.w3.org/2003/06/sw-vocab-status/ns#> .
+@prefix vann: <http://purl.org/vocab/vann/> .
+
+<http://www.w3.org/ns/solid/notifications#>
+    a owl:Ontology ;
+    dc:license <https://creativecommons.org/publicdomain/zero/1.0/> ;
+    rdfs:label "Solid Notifications"@en ;
+    rdfs:comment "The vocabulary used by the Solid Notification protocol specification."@en ;
+    rdfs:isDefinedBy <http://www.w3.org/ns/solid/notifications#> ;
+    rdfs:seeAlso <https://solidproject.org/TR/notifications-protocol>
+    dc:issued "2022-01-04"^^xsd:date ;
+    vann:preferredNamespacePrefix "notify" ;
+    vann:preferredNamespaceUri "http://www.w3.org/ns/solid/notifications#"^^xsd:anyURI .
+
+# ---------
+#  Classes
+# ---------
+notify:WebSocketSubscription2021
+    a rdf:Class ;
+    rdfs:label "WebSocket Subscription 2021"@en ;
+    rdfs:comment "A Subscription type that uses the WebSocket API"@en ;
+    rdfs:isDefinedBy <https://solidproject.org/TR/websocket-subscription-2021> ;
+    vs:term_status "testing" .
+
+# ------------
+#  Properties
+# ------------
+notify:accept
+    a rdf:Property ;
+    rdfs:label "accept"@en ;
+    rdfs:comment "A MIME-Type value that indicates the desired serialization of the notification."@en ;
+    rdfs:isDefinedBy <https://solidproject.org/TR/notifications-protocol#feature-accept> ;
+    vs:term_status "testing" .
+
+notify:expiration
+    a rdf:Property ;
+    rdfs:label "expiration"@en ;
+    rdfs:comment "An ISO 8601 datetime value indicating a proposed expiration point for a subscription."@en ;
+    rdfs:isDefinedBy <https://solidproject.org/TR/notifications-protocol#feature-expiration> ;
+    vs:term_status "testing" .
+
+notify:feature
+    a rdf:Property ;
+    rdfs:label "feature"@en ;
+    rdfs:comment "A property used to describe the features supported by a particular notification channel."@en ;
+    rdfs:isDefinedBy <http://www.w3.org/ns/solid/notifications#> ;
+    vs:term_status "testing" .
+
+notify:notificationChannel
+    a rdf:Property ;
+    rdfs:label "notification channel"@en ;
+    rdfs:comment "A property used to connect a Notification Subscription Metadata Resource to a particular notification channel."@en ;
+    rdfs:isDefinedBy <http://www.w3.org/ns/solid/notifications#> ;
+    vs:term_status "testing" .
+
+notify:rate
+    a rdf:Property ;
+    rdfs:label "rate"@en ;
+    rdfs:comment "An ISO 8601 duration value indicating the minimum amount of time to elapse between notifications sent to the client."@en ;
+    rdfs:isDefinedBy <https://solidproject.org/TR/notifications-protocol#feature-rate> ;
+    vs:term_status "testing" .
+
+notify:source
+    a rdf:Property ;
+    rdfs:label "source"@en ;
+    rdfs:comment "The IRI of a resource used by a client to establish a connection to a notification channel."@en ;
+    rdfs:isDefinedBy <https://solidproject.org/TR/websocket-subscription-2021#websocketsubscription2021-source> ;
+    vs:term_status "testing" .
```

Comment:
The description is generally fine but perhaps we should introduce this in the Notifications Protocol (and refer to it with isDefinedBy here) so that some subscription types could make use of it.